engine/src/main/java/org/camunda/bpm/engine/history/HistoricIncident.java

```diff
@@ -150,4 +150,9 @@
 
   /** The time the historic incident will be removed. */
   Date getRemovalTime();
+
+  /**
+   * Returns the id of the activity on which the last exception occurred.
+   */
+  String getLastFailingActivityId();
```

Comment:
Why is named "Last...."? Isn't it so that an incident is created because of exactly one exception? Hence there will be exactly one activityId, and hence there will be no "first" or "last" one.

Reply:
Good point! Will rename that to `failedActivityId()` 👍